src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {
+  Popover as CorePopover,
+  PopoverProps as CorePopoverProps,
+} from 'wix-ui-core/popover';
+
+import { TPAComponentProps } from '../../types';
+import { generateKey } from '../../common/random';
+import { KEYS } from '../../common/keyCodes';
+
+import { TriggerAction } from './constants';
+import { st, classes } from './Popover.st.css';
+
+export interface PopoverProps
+  extends TPAComponentProps,
+    Omit<CorePopoverProps, 'shown'> {
+  triggerAction?: TriggerAction;
+  wiredToSiteColors?: boolean;
+}
+
+export const Popover: React.FC<PopoverProps> & {
+  Element?: React.FC;
+  Content?: React.FC;
+} = props => {
+  const {
+    triggerAction,
+    wiredToSiteColors,
+    children,
+    className,
+    ...rest
+  } = props;
+  const [isShown, setIsShown] = React.useState<boolean>(false);
+  const [id] = React.useState(generateKey('popover'));
+
+  const [Element, Content] = React.Children.toArray(
+    children,
+  ) as React.ReactElement[];
```

Comment:
This means the `Element` child **has** to be written above the `Content` child, no?
Maybe it would be better to iterate over the children and check their type.
You can see how it was done in the AvatarGroup component:
https://github.com/wix/wix-ui-tpa/blob/3a76e4002d818cd59d0a00ba2589b900bd4b6c24/src/components/AvatarGroup/AvatarGroup.tsx#L47